packer/vault-policies/write-node-policies.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import json
+import sys
+
+NETWORK_ID = ''
+AWS_ACCOUNT_ID = ''
+SETUP_FILE = ''
+OUTPUT_DIR = ''
+
+def policy_string(region, role, role_index, cluster_index):
+    if role == "bootnodes":
+        return """\
```

Comment:
Maybe use single quotes to delineate the strings to save a bunch of escaping? It'd be easier to read if it works out.

Reply:
Unfortunately, HCL [requires all strings to be double-quoted](https://www.terraform.io/docs/configuration/syntax.html#terraform-syntax).  Agreed that it makes this chunk of code illegible as all hell.